.prettierrc.js

```diff
@@ -0,0 +1,15 @@
+// Documentation for this file: https://prettier.io/docs/en/configuration.html
+
+/** @type {import('prettier').Options} */
+module.exports = {
+  // Use .gitattributes to manage newlines
+  endOfLine: "auto",
+
+  // Avoid line breaks in markdown
+  proseWrap: "never",
+
+  // Disabling search dirs and pointing directly at plugins
+  // makes it easier for VSCode and WebStorm to find them.
```

Comment:
The latter is probably not true in our case so let's shorten this
```suggestion
  // Disabling search dirs and pointing directly at plugins to keep configuration explicit
```